lib/common/common.go

```diff
@@ -22,8 +22,13 @@ import (
 )
 
 const (
-	defaultTag    = "latest"
-	schemaVersion = "0.5.1"
+	defaultTag                = "latest"
+	schemaVersion             = "0.5.1"
+	AppcDockerV1IndexURL      = "appc.io/docker/v1/indexurl"
+	AppcDockerV1Repository    = "appc.io/docker/v1/repository"
+	AppcDockerV1Tag           = "appc.io/docker/v1/tag"
+	AppcDockerV1ImageID       = "appc.io/docker/v1/imageid"
+	AppcDockerV1ParentImageID = "appc.io/docker/v1/parentimageid"
```

Comment:
I was thinking to export them for use by rkt but this doesn't make much sense. I'll make them unexported.